src/v2/Apps/Home/Components/HomeArtworkModules.tsx

```diff
@@ -10,6 +10,8 @@ import { MyBidsQueryRenderer } from "v2/Apps/Auctions/Components/MyBids/MyBids"
 import { HomeFeaturedShowsLazyQueryRenderer } from "./HomeFeaturedShows"
 import { compact } from "lodash"
 
+const EXCLUDED_MODULES = ["popular_artists", "generic_gene"]
```

Comment:
The query for this supports an `exclude` argument. So we can just adjust the query:

```graphql
artworkModules(exclude: [POPULAR_ARTISTS, GENERIC_GENES]) {
  ...
}
```